src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1021,8 +1022,9 @@ def run_dm_activities_with_ior(self, tool, pool, cont, create_dataset=False):
             pool2 = self.get_pool()
             # Use dfuse as a shared intermediate for serialize + deserialize
             dfuse_cont = self.get_container(pool, oclass=self.ior_cmd.dfs_oclass.value)
-            self.start_dfuse(self.dfuse_hosts, pool, dfuse_cont)
-            self.serial_tmp_dir = self.dfuse.mount_dir.value
+            dfuse = get_dfuse(self, self.dfuse_hosts)
+            start_dfuse(self, dfuse, pool, dfuse_cont)
+            self.serial_tmp_dir = dfuse.mount_dir.value
```

Comment:
Maybe need to continue using `self.dfuse`

Reply:
Done